gradle/libs.versions.toml

```diff
@@ -69,6 +69,6 @@ plugin-android = { module = "com.android.tools.build:gradle", version.ref = "agp
 plugin-dokka = { module = "org.jetbrains.dokka:dokka-gradle-plugin", version.ref = "kotlin" }
 plugin-grgit = { module = "org.ajoberstar.grgit:grgit-gradle", version = "5.0.0" }
 plugin-kotlin = { module = "org.jetbrains.kotlin:kotlin-gradle-plugin", version.ref = "kotlin" }
-plugin-mavenPublish = { module = "com.vanniktech:gradle-maven-publish-plugin", version = "0.21.0" }
+plugin-mavenPublish = { module = "com.vanniktech:gradle-maven-publish-plugin", version = "0.24.0" }
```

Comment:
The `paparazzi-sources.jar` contents are relocated into `main/` and only after that the package structure starts: `main/app/cash`. This is unusual I think.

`paparazzi-gradle.plugin-sources.jar` is starts with `app/cash` on both main and this PR.

Reply:
The additional index.html and paparazzi.js files are now missing from the `-sources.jar`.

Reply:
Same applies to `paparazzi-agent-sources.jar`

Reply:
>The paparazzi-sources.jar contents are relocated into main/ and only after that the package structure starts: main/app/cash. This is unusual I think.
>paparazzi-gradle.plugin-sources.jar is starts with app/cash on both main and this PR.

This is how the Kotlin plugin creates the sources jar (previously we had a custom task for creating sources in the publish plugin), I guess they share the `kotlinSourcesJar` task with multiplatform where you need to differentiate between source sets because otherwise you might have the same file twice. IntelliJ/Studio seem to be fine in handling this. The reason why the Gradle plugin is different is that Gradle is creating the sources jar there.

> The additional index.html and paparazzi.js files are now missing from the -sources.jar.

Also seems to come from now using `kotlinSourcesJar`. Personally I don't think this isn't a problem because resources are included in the main jar already but if it is I can see if we can workaround this.